app/controllers/user_sessions_controller.rb

```diff
@@ -6,7 +6,11 @@ def new
   def create
     if @user = login(params[:email], params[:password])
       if @user.is_student?
-        redirect_back_or_to(:assignments) #, notice: 'Login successful')
+        if Time.now <= $studio_due && current_user.studio.nil?
+          redirect_to edit_student_path(current_user)
```

Comment:
Same here, add a flash notice before the redirect_to call to let the user know they need to sign up for a studio before proceed to the assignment page.